### ISSUE TITLE
Use streamlink instead of youtube-dl

### ISSUE THREAD

**Issue by ghost** [oliverjrose99/Recordurbate#39]
Hello.

Is there any way I can use streamlink instead of youtube-dl? It generally seems more stable for this kind of thing, and it allows me to view a file while it is downloading. That way, if I happen to miss the beginning of a stream, I can open the file and rewind while it's still recording. I was able to find a streamlink chaturbate plugin <a href="https://github.com/jbanquo/chaturbate-downloader-v2/blob/master/chaturbate.py">here</a> and <a href="https://github.com/Brant-Salamon/streamlink.plugins.chaturbate/blob/master/chaturbate.py">here</a>, and the plugin worked when I tested it.

Let me know if this would be possible.
Thanks.

**Comment by botnet88** [comment]
I have been messing with the program for a few hours, and I was able to have this work with streamlink. I uploaded the changes I made <a href="https://github.com/botnet88/Recordurbate-streamlink">here</a>. The only issue I have now is the file name that gets written. I made the program write to videos/username-date.mp4, and with htop I can see that it is doing it:
<br>
![htop](https://user-images.githubusercontent.com/61393492/75124010-e87caf00-5671-11ea-99bd-3ad50170733b.png)
Using the`ls`command also shows that the file name is correct.
<br>
However, when I access the file on Windows, this is what is shown:
![name-issue](https://user-images.githubusercontent.com/61393492/75124059-4f9a6380-5672-11ea-9e9f-01f9c33d12b0.png)

Is there any way to fix the naming problem?
